README.md

```diff
@@ -379,6 +379,38 @@ const server = kx.server({
 })
 ```
 
+### `crypto.kdf.keygen(key)` <a name="kdf-keygen"></a>
+
+> **Stability: 2** - Stable
+
+Generate a key of length `crypto_kdf_KEYBYTES`.
+
+```js
+const { crypto_kdf_KEYBYTES } = require('ara-crypto/sodium')
```

Comment:
The user of this module should not need to interact with sodium directly. the `keygen` function here should be functional. That is, the function allocates the buffer for the caller, and returns it.

Reply:
Could you advertise the functional approach for key generation?

Reply:
Yes, I'll make the change.